include/acvp/acvp_lcl.h

```diff
@@ -1143,12 +1154,17 @@ typedef struct acvp_kdf108_mode_params {
     ACVP_SL_LIST *counter_lens;
     int empty_iv_support;
     int requires_empty_iv;
+    ACVP_JSON_DOMAIN_OBJ derivation_keylens;
```

Comment:
Domain objects now need (its a newer change) to be freed explicitly in acvp.c using acvp_cap_free_domain. Thanks!

Reply:
I posted a fix for this with the scope of the kdf108 algorithm.  I don't believe the hash LDTs used any domain-type parameters.